docs/06-concepts/19-experimental.md

```diff
@@ -24,6 +24,10 @@ The current options you can pass are:
 
 Inheritance allows you to define class hierarchies in your model files by sharing fields between parent and child classes, simplifying class structures and promoting consistency by avoiding duplicate field definitions.
```

Comment:
Suggestion: Add a warning that a new subtype of a class might be a breaking change with older clients.

Regardless of how we solve the serialization/deserialization issue, this is going to be a fact. So we might as well add it as a warning to the documentation.